cliquet/tests/test_views_errors.py

```diff
@@ -153,3 +153,30 @@ def test_querystring_is_preserved_during_redirection(self):
         response = self.app.get('/home/articles?_since=42')
         self.assertEqual(response.location,
                          'http://localhost/v0/home/articles?_since=42')
+
+
+class TrailingSlashRedirectViewTest(BaseWebTest, unittest.TestCase):
+    def test_redirect_to_non_trailing_slash_version(self):
+        # GET on the hello view.
+        response = self.app.get('/')
+        self.assertEqual(response.status_int, 200)
+
+        # GET on the fields view.
```

Comment:
I would that here another test method starts:  test_it_redirects_if_ends_with_slash